tests/manual-tests/lua-cli.js

```diff
@@ -24,10 +26,37 @@ for (;;) {
     if (input.length === 0)
         continue;
 
-    let buffer = lua.to_luastring(input);
-    if (lauxlib.luaL_loadbuffer(L, buffer, buffer.length, lua.to_luastring("=stdin"))
-        || lapi.lua_pcall(L, 0, 0, 0)) {
+    let status;
+    {
+        let buffer = lua.to_luastring(input);
+        status = lauxlib.luaL_loadbuffer(L, buffer, buffer.length, stdin);
+    }
+    if (status !== lua.thread_status.LUA_OK) {
+        lapi.lua_pop(L, 1);
+        let retbuffer = lua.to_luastring("return " + input);
```

Comment:
Then it was either valid syntax, or we try adding a second 'return' (same as what normal lua interpreter does)